### PR TITLE
Filter out dependabot PRs from webhooks

### PR DESCRIPTION
**Pull request by tsickert**
Need to add `WEBHOOK_URL` as repository secret in order to validate this.

### CODE REVIEW

File: .github/workflows/pull-request-filter.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  notify_if_not_dependabot:
+    if:  ${{ github.actor }} != 'dependabot'
+    runs-on: ubuntu-latest
+    steps:
+    - name: Save event to file
+      run: echo ${{ github.event }} > event.json
+    - name: Discord Webhook Action
+      uses: tsickert/discord-webhook@v5.0.0
+      with:
+        webhook-url: ${{ secrets.WEBHOOK_URL }}
```
Comment (markphelps) [inline .github/workflows/pull-request-filter.yml:20]: could we rename this something like `secrets.DISCORD_WEBHOOK_URL` or `secrets.DISCORD_PR_WEBHOOK_URL` so its more clear what the secret is for when looking over all the secrets?

**Review by markphelps — COMMENTED** [review]
this is awesome! one small request

File: .github/workflows/pull-request-filter.yml (hunk 1 above)
Comment (tsickert) [reply .github/workflows/pull-request-filter.yml]: Good call, updated!

**Review by markphelps — APPROVED** [review]
🤘

**Pull request merged**